unity-client/Assets/Scripts/Tests/IntegrationTestSuite_Legacy.cs

```diff
@@ -49,6 +51,14 @@ protected virtual IEnumerator SetUp()
 
         runtimeGameObjectsRoot = new GameObject("_RuntimeGameObjectsRoot");
 
+        testSceneIntegrityChecker = new TestSceneIntegrityChecker();
+
+        //NOTE(Brian): integrity checker is disabled in batch mode to make it run faster in CI
+        if (Application.isBatchMode || !enableSceneIntegrityChecker)
+        {
+            testSceneIntegrityChecker.enabled = false;
+        }
```

Comment:
If we don't use the integrity checker in the CI when do we use it? XD

Reply:
I suppose we still use local test runs 😥

Reply:
I'm thinking that maybe we can try to enable it in CI right now, as we managed to reduce the test run time.